### PR TITLE
feat: support to customize the cmd path and args

### LINKED ISSUE #6: Windows built-in ssh client doesn't work
FYI: If using the Windows built-in ssh client, the `SnipsCreate` doesn't work. When I change to use cygwin ssh client, it works correctly.

How to remove built-in ssh client:

> Open Settings, then go to Apps > Apps & Features. Go to Optional Features. In the list, select OpenSSH Client or OpenSSH Server. Select Uninstall.

### PR DESCRIPTION
**Pull request by Pagliacii**
Close #6 

```lua
local opts = {
  post_behavior = "echo",  -- or "yank"
  cat_cmd = "cat",
  ssh_cmd = "ssh",
}
require("snips.nvim").setup(opts)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ The simple idea behind this plugin is to:
 ```lua
 local opts = {
   post_behavior = "echo",  -- or "yank"
+  cat_path = "cat",
+  cat_args = nil,
+  ssh_path = "ssh",
+  ssh_args = nil,
```
Comment (Sanix-Darker) [inline README.md]: why not just `cat_cmd` and `ssh_cmd` ? i found this format too verbose... 🤔 
```suggestion
  cat_cmd = "cat -e ", -- can be bat or less
  ssh_cmd = "ssh -B ", -- choose your custom ssh client
```

What do you think ?

**Review by Sanix-Darker — COMMENTED** [review]
Thank you very much for this Pull Request mate 🙏🏽 🔥 , honestly, you rock !

I was having a small suggestion, and have your point of view !
apart for that, everything LGTM !
Once we're set on and sync on the question, I will approve/merge.

**Comment by Pagliacii** [comment]
Updated

**Review by Sanix-Darker — APPROVED** [review]
YOU ROCK 🔥 🚀

**Pull request merged**